setup.py

```diff
@@ -13,6 +13,7 @@
 
 
 REQUIREMENTS = [
+    'Django>=1.6,<1.8',
```

Comment:
we have explicit requirements here https://github.com/aldryn/aldryn-jobs/pull/62/files#diff-2eeaed663bd0d25b7e608891384b7298R36

so this line shouldn't be here.